packages/solidity/src/ics26.rs

```diff
@@ -29,6 +30,37 @@ impl IICS26RouterMsgs::Packet {
         path
     }
 
+    /// Returns the packet commitment
+    #[must_use]
+    pub fn commit_packet(&self) -> Vec<u8> {
```

Comment:
Should be renamed imo.
```suggestion
    pub fn packet_commitment(&self) -> Vec<u8> {
```

Reply:
I kinda of agree, but I was using the naming from ibc-go to just be consistent. I'm OK with either honestly